packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -592,25 +600,56 @@ export function MainStackScreen() {
       pincodeType,
       account,
       hasSeenVerificationNux,
+      shouldSkipOnboardingEducationScreen,
```

Comment:
I think the code could be easier to understand and safer from bugs if this were named differently, like `savedShouldSkipEducationScreen`. That way there's less ambiguity over whether you should use this or the underscore-prefixed variable in the actual logic later on

Reply:
so you could have this as 
```suggestion
      shouldSkipOnboardingEducationScreen: savedShouldSkipEducationScreen,
```
and then refer to it as `savedShouldSkipEducationScreen` from here on